tools/evasion/evasion_common/shellcode_help.py

```diff
@@ -20,13 +20,12 @@
 try:
     sys.path.append("/etc/veil/")
     import settings
-
 except ImportError:
-    print("\n [!] ERROR #1: Run %s\n" % (os.path.abspath("./config/update-config.py")))
+    print( "\n [!] ERROR #1-4: Can't import /etc/veil/settings.py.   Run: %s\n" % ( os.path.abspath( "./config/update-config.py" ) ) )
     sys.exit()
 
 
-sys.path.insert(0, settings.VEIL_EVASION_PATH + 'tools/ordnance')
+sys.path.insert(0, settings.VEIL_PATH + 'tools/ordnance')
 import tool as ordnance_import
```

Comment:
This should be
`import tools as ordnance_import`
rather than
`import tool as ordnance_import `since the module(folder) name is tools

Reply:
But the file it's including is called `./tools/ordnance/tool.py' not `./tools/ordnance/tools.py' ?